tests/Api/Admin/AdminUsersTest.php

```diff
@@ -41,4 +43,42 @@ public function it_allows_admin_users_to_log_in(): void
 
         $this->assertResponse($response, 'admin/log_in_admin_response', Response::HTTP_OK);
     }
+
+    /** @test */
+    public function it_allows_admin_user_to_change_timezone(): void
+    {
+        $adminUser = $this->loadFixturesAndGetAdminUser();
+        $header = $this->getLoggedHeader();
+
+        $this->client->request(
+            'PUT',
+            sprintf('/api/v2/admin/administrators/%s', $adminUser->getId()),
+            [],
+            [],
+            $header,
+            json_encode([
+                'timezone' => 'Europe/Paris'
+            ])
+        );
+
+        $response = $this->client->getResponse();
+
+        $this->assertResponse($response, 'admin/administrators/change_timezone_admin_response', Response::HTTP_OK);
+    }
+
+    private function loadFixturesAndGetAdminUser(): AdminUserInterface
```

Comment:
I believe we can skip this private function and use `loadFixturesFromFile` method directly in the test function. 1 more line there - 6 less lines here :dancer: